firestore.rules

```diff
@@ -1,7 +1,10 @@
 service cloud.firestore {
   match /databases/{database}/documents {
+ 	 match /{collectionName}/{documentId} {
+  	 allow read, write : if collectionName == "settings";
```

Comment:
Bør kanskje ha ein sjekk på `auth.uid` for å unngå at settings er heilt åpent?

Reply:
Oida! Ja, det er en god ide 😅